torch/fx/experimental/partitioner_utils.py

```diff
@@ -71,6 +71,7 @@ class PartitionerConfig(NamedTuple):
     devices: List[Device]
     is_sparse_nn: bool = False
     is_cost_aware: bool = False
+    is_kl: bool = False
```

Comment:
Are these three options mutually exclusive? If so, I suggest defining an enum to select which mode to use rather than having separate bools

Reply:
It is mutually exclusive. An enum is a good idea